lib/src/blaze/trial_decryptions.rs

```diff
@@ -169,7 +169,7 @@ impl TrialDecryptions {
 
         for compact_block in compact_blocks {
             let height = BlockHeight::from_u32(compact_block.height as u32);
-            info!("trial decrypting block {}", height);
+            //info!("trial decrypting block {}", height);
```

Comment:
Instead of commenting this out...   should we change its "loudness"?

Reply:
Honestly I don't think we need it.. Happy to just cut it entirely